src/windows/Link.tsx

```diff
@@ -0,0 +1,150 @@
+/**
+* Link.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* RN Desktop-specific implementation of the cross-platform Link abstraction.
+*/
+
+import React = require('react');
+import RNW = require('react-native-windows');
+import {applyFocusableComponentMixin, FocusManagerFocusableComponent} from '../native-desktop/utils/FocusManager';
+import PropTypes = require('prop-types');
+
+import EventHelpers from '../native-common/utils/EventHelpers';
+import {Link as LinkCommon } from '../native-common/Link';
+
+const KEY_CODE_ENTER = 13;
+const KEY_CODE_SPACE = 32;
+
+const DOWN_KEYCODES = [KEY_CODE_SPACE, KEY_CODE_ENTER];
+const UP_KEYCODES = [KEY_CODE_SPACE];
+
+export interface LinkContext {
+    isRxParentAText?: boolean;
+}
+
+// Simple check for the presence of the updated React Native for Windows
+const HasFocusableWindows = (RNW.FocusableWindows !== undefined);
+
+export class Link extends LinkCommon implements FocusManagerFocusableComponent {
+    static contextTypes: React.ValidationMap<any> = {
+        isRxParentAText: PropTypes.bool,
+    };
+    context: LinkContext;
+
+    private _focusableElement : RNW.FocusableWindows | null = null;
+
+    private _onFocusableRef = (btn: RNW.FocusableWindows | null): void => {
+        this._focusableElement = btn;
+    }
+
+    render() {
+
+        // Fallback to native-common fast if the keyboard enabled component is not available
+        if (!HasFocusableWindows) {
+            return super.render();
+        }
+
+        let content: JSX.Element = super.render();
+
+        // The "in text parent" case requires a special nyi control.
```

Comment:
>// The "in text parent" case requires a special nyi control. [](start = 8, length = 60)

Is this something we plan on doing later? If so, I think we should create a VSTS task for it and prefix this comment with `TODO` and reference the VSTS task number.

Also, I think it would be clearer if you created the branch for this case and then put the comment in that otherwise empty branch.

Reply:
Yes, it's the Xaml Link we talked about.
I think the task exists, yet adding the # here would be not very relevant to outside Msft, that's why I avoided it.  

I don't get the branch comment.

---
In reply to: [159358659](https://github.com/Microsoft/reactxp/pull/404#discussion_r159358659) [](ancestors = 159358659)

Reply:
What I meant is to write something like this:

```
if (this.context && !this.context.isRxParentAText) {
  /* a bunch of implementation details that you have */
} else if (this.context && this.context.isRxParentAText) {
  // TODO: The "in text parent" case requires a React Native view that maps to XAML Hyperlink but this RN view isn't implemented yet.
}
```

---
In reply to: [159486509](https://github.com/Microsoft/reactxp/pull/404#discussion_r159486509) [](ancestors = 159486509,159358659)